templates/seshdash/high_chart_line_bar.html

```diff
@@ -0,0 +1,103 @@
+ {% block high_chart_line_bar %}
+
+    <div class="col-lg-12">
+        <div class="panel panel-default">
+                <div class="panel-heading">
+                    <!-- TODO make this a variable -->
+                    <i class="fa fa-bar-chart-o fa-fw"></i>
+                    Daily PV Production with Cloud cover forecast In High Chart
+                </div>
+            <div class="panel-body" id="containerhigh">
+
+
+            </div>
+        </div>
+    </div>
+
+<script language="JavaScript">
+ $(function () {
```

Comment:
Please move all Javascript to the dashboard javascript file. 

There should only be minimal javascript in the template so let's avoid it if we can. You can create the data objects using the template. and then have the javascript in a seperate file talk to that.